probe-rs/Cargo.toml

```diff
@@ -150,15 +155,14 @@ figment = { version = "0.10", features = [
 ], optional = true }
 goblin = { version = "0.8", optional = true }
 indicatif = { version = "0.17", optional = true }
-insta = { version = "1", features = ["yaml", "filters"] }
+insta = { version = "1", features = ["yaml", "filters"], optional = true }
```

Comment:
Why do we need this as a non dev dep in the first place?

Reply:
It's used in https://github.com/probe-rs/probe-rs/blob/b329ea15ba32c196c83e01504fa5644c6dc320af/cli/src/bin/probe-rs/cmd/dap_server/debug_adapter/dap/repl_commands.rs#L212

Reply:
sorry I was trying to fix up the link that was 404 for me and we raced on it it seems